docker-compose.yaml

```diff
@@ -31,5 +47,33 @@ services:
       KC_HOSTNAME_STRICT: false
       KC_HTTPS_KEY_STORE_FILE: /etc/keycloak-keystore.p12
       KC_HTTPS_KEY_STORE_PASSWORD: changeit
+      KC_HEALTH_ENABLED: true
       KEYCLOAK_ADMIN: admin
       KEYCLOAK_ADMIN_PASSWORD: admin
+    restart: on-failure
+    healthcheck:
+      test: ['CMD', 'curl', '-k', '-f', 'https://localhost:10443/q/health/ready']
+      start_period: 30s
+      interval: 10s
+
+  vp-server:
```

Comment:
Can we add an option to choose whether the vp-server is started? Maybe, [Compose Profiles](https://docs.docker.com/compose/profiles/) are suitable for this.

Reply:
I think profiles are the opposite functionality of what we want. By default you'd want everything to run. You can run `docker compose up ibm-fhir keycloak` to only run those services.

Reply:
Right, I totally forgot about that. Sorry.